server/events/models/models.go

```diff
@@ -343,6 +343,8 @@ type ProjectCommandContext struct {
 	// Workspace is the Terraform workspace this project is in. It will always
 	// be set.
 	Workspace string
+	// Additional env variables
+	Env map[string]string
```

Comment:
I don't think we should add this here because I like how this struct is fully filled out by the builder prior to passing it to the project command runner. Instead I think we can just use a local variable inside the project_command_runner

Reply:
Hm... but how we can use this variable in every runner? should we send it as an argument? like:
`p.InitStepRunner.Run(p.InitStepRunner.Run(ctx, step.ExtraArgs, absPath, envs)`?

Or maybe set envs form CLI flag for example. 
`atrantis server --env SOME=global_envs --env ANOTHER=value` // but  we can do that like simple global variables : )

Reply:
I was thinking as an arg to each runner.